### PR TITLE
Dev ecmas6

### PR DESCRIPTION
**Pull request by lucasmelul**
## Summary
Finished Card Dev EcmaScript6
## Trello Card
https://trello.com/c/cHbNAbQF/20-dev-stack-ecmascript-6

### CODE REVIEW

File: wbooks/app/app.js [hunk 1]
```diff
@@ -46,21 +46,21 @@ angular.module('wbooks').config([
 angular.module('wbooks').controller("HomeController", [
   '$http', 
   function($http) {
-    var homeCtrl = this;
+    let homeCtrl = this;
```
Comment (mcallegari10) [inline wbooks/app/app.js]: In ES6 you don't need to do this anymore :smile: Luckily fir us, JavaScript now support self awareness so when you refer to `this` inside an anonymous function or a method of the controller, you can refer to the Controller as `this`, so no need for this assignment :smile:

File: wbooks/app/app.js [hunk 2]
```diff
@@ -46,21 +46,21 @@ angular.module('wbooks').config([
...
     homeCtrl.optFilter = [
       { id: '0', label: 'Seleccione un filtro' },
       { id: 'title', label: 'Titulo' },
       { id: 'author', label: 'Autor' }
     ];
 
-    $http.get("/books.json",[]).success(function(data) {
-      homeCtrl.books = data;
+    $http.get("/books.json",[]).success((data) => {
+      home.books = data;
```
Comment (mcallegari10) [inline wbooks/app/app.js]: `this` instead of home

File: wbooks/app/app.js [hunk 3]
```diff
@@ -46,21 +46,21 @@ angular.module('wbooks').config([
...
     });
 
-    homeCtrl.filterList = function() {
+    homeCtrl.filterList = () => {
```
Comment (mcallegari10) [inline wbooks/app/app.js]: `this` instead of homeCtrl

File: wbooks/app/app.js [hunk 4]
```diff
@@ -78,14 +78,14 @@ angular.module('wbooks').controller("HomeController", [
 angular.module('wbooks').controller("BookDetailController", [
   '$http','$stateParams',
   function($http, $stateParams) {
-    var bookDetail = this;
+    let bookDetail = this;
```
Comment (mcallegari10) [inline wbooks/app/app.js]: Same as before, no need for this :smile:

File: wbooks/app/app.js [hunk 5]
```diff
@@ -78,14 +78,14 @@ angular.module('wbooks').controller("HomeController", [
...
     bookDetail.bookId = $stateParams.bookId;
 
-    $http.get("/books.json",[]).success(function(data) {
+    $http.get("/books.json",[]).success((data) => {
       bookDetail.book = data.filter(bookDetail.getById)[0];
     });
 
-    bookDetail.getById = function(item) {
+    bookDetail.getById = (item) => {
```
Comment (mcallegari10) [inline wbooks/app/app.js]: `this` instead of `bookDetail`

File: wbooks/app/lib/app.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+		url: '/',
+		templateUrl: 'templates/home.html'
+	}).state('book_detail', {
+		name: 'book_detail',
+		url: '/books/:bookId',
+		templateUrl: 'templates/book_detail.html'
+	});
+});
+
+app.controller("HomeController", ['$http', function ($http) {
```
Comment (mcallegari10) [inline wbooks/app/lib/app.js]: There are two `app.js`, beware of this. Please check this :smile:

File: wbooks/app/scss/base.scss [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  background-color: $color-white;
+  font-family: $font-primary;
+}
+
+.cont-page {
```
Comment (SKOLZ) [inline wbooks/app/scss/base.scss]: this was fixed in the previous pr, rebase and reasign.

**Pull request merged**